src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -100,6 +101,7 @@
 		public static CustomJoystick operatorStick;
 		public static TeensyController teensyStick;
 		public static PDP pdp;
+		public static PowerDistributionPanel CANPDPlogger;
```

Comment:
As soon as we have better brownouts finished, you will be able to get everything except temperature from the PDP. This is really, really preferable as PowerDistributionPanel throws unstoppable exceptions if the PDP is disconnected, which would kill robot code during a match.
Also, if you need temperature, I can add it too.